### PR TITLE
[MIG] 10.0 super calendar

### PR DESCRIPTION
**Pull request by jeanpaulrobineau**
Hello,
I have ported this super calendar for version 10.

Kind regards,
Jean-Paul Robineau

### CODE REVIEW

**Comment by mstuttgart** [comment]
LGTM :+1: 

@jeanpaulrobineau please, fix the *Flake8* erros in Travis IC: https://travis-ci.org/OCA/server-tools/jobs/177931147#L322

**Review by eLBati — CHANGES_REQUESTED** [review]
Thanks @jeanpaulrobineau 
looking at the diff with 8.0 (last release of super_calendar) with
```git diff 8.0 apertoso/10.0_super_calendar super_calendar/```
I see some commits from 8.0 are missing.
Please follow the procedure described at
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0#technical-method-to-migrate-a-module-from-90-to-100-branch
but replacing ```9.0``` with ```8.0```

If you need help, please join #oca channel at freenode.net

**Comment by ovnicraft** [comment]
@jeanpaulrobineau can you update this PR ?

**Comment by JosDeGraeve** [comment]
Hi,

I worked togheter with Jean-Paul on this.  Afaik, these issues have been fixed:

* CLA for @jeanpaulrobineau 
* Travis Flake8 error
* missing commits from 8.0

**Comment by ovnicraft** [comment]
so we wait for @eLBati review

**Review by eLBati — APPROVED** [review]
thanks

**Comment by jeanpaulrobineau** [comment]
Hello,

When can this be merged as everything seems ok now?

Kind regards 
Jean-Paul Robineau

**Review by JosDeGraeve — APPROVED** [review]
Been in production for a while now at some of our customers.

**Review by Ehtaga — APPROVED** [review]

**Comment by Ehtaga** [comment]
@gurneyalex @dreispt @guewen Can we merge this PR?

**Review by mstuttgart — APPROVED** [review]

File: super_calendar/models/super_calendar.py
Comment (elicoidal) [inline super_calendar/models/super_calendar.py:28]: Could you please replace with short headers in all py files following: https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py

File: super_calendar/README.rst
Comment (elicoidal) [inline super_calendar/README.rst:26]: Could you please update the README file to the newest template available here: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: super_calendar/__manifest__.py
Comment (elicoidal) [inline super_calendar/__manifest__.py:32]: update the URL to the repo URL

File: super_calendar/__manifest__.py
Comment (elicoidal) [inline super_calendar/__manifest__.py:47]: remove empty keys

File: super_calendar/views/super_calendar_view.xml
Comment (elicoidal) [inline super_calendar/views/super_calendar_view.xml:3]: remove data tag

**Review by elicoidal — CHANGES_REQUESTED** [review]
Thanks for the contribution.
Could you please take the opportunity to update standard information such as README, manifest and copyright headers?

**Comment by ddufresne** [comment]
@jeanpaulrobineau are you still working on this? Can you make the few changes asked by @elicoidal so that we can move forward?

**Pull request merged**
